eng/pipelines/source-build-sdk-diff-tests.yml

```diff
@@ -26,6 +26,7 @@ jobs:
     targetRid: centos.8-x64
     architecture: x64
     dotnetDotnetRunId: ${{ parameters.dotnetDotnetRunId }}
+    excludeArtifactsSize: false

```

Comment:
I agree that this makes more sense. Is there a reason that the OmniSharpTests function as "exclude" rather than "include"? The reason I implemented the logic this way was to flow with the style of "excludeOmniSharpEnv"

Reply:
Addressed in [this commit](https://github.com/dotnet/installer/pull/17233/commits/8ec128e7e0e58aa70e87b705d20616b29250bb04).